pallets/liquidity-pools/src/lib.rs

```diff
@@ -800,22 +804,14 @@ pub mod pallet {
 				Error::<T>::TrancheNotFound
 			);
 
-			ensure!(
-				T::Permission::has(
-					PermissionScope::Pool(pool_id),
-					who,
-					Role::PoolRole(PoolRole::PoolAdmin)
-				),
-				Error::<T>::NotPoolAdmin
-			);
```

Comment:
Why this?

Reply:
Feature creeping. That is a change needed for letting who pay and remove need for admin then